ci-operator/step-registry/telcov10n/functional/cnf-network/cnf-config/telcov10n-functional-cnf-network-cnf-config-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+echo "Create group_vars directory"
+mkdir /eco-ci-cd/inventories/ocp-deployment/group_vars
+
+echo "Copy group inventory files"
+cp ${SHARED_DIR}/all /eco-ci-cd/inventories/ocp-deployment/group_vars/all
+cp ${SHARED_DIR}/bastions /eco-ci-cd/inventories/ocp-deployment/group_vars/bastions
```

Comment:
small nit - The path is reused throughout the code best move it to a variable

```suggestion
ECO_CI_PATH="/eco-ci-cd/inventories/ocp-deployment"

echo "Create group_vars directory"
mkdir "${ECO_CI_PATH}/group_vars

echo "Copy group inventory files"
cp ${SHARED_DIR}/all "${ECO_CI_PATH}/group_vars/group_vars/all"
cp ${SHARED_DIR}/bastions "${ECO_CI_PATH}/group_vars/bastions"
```

Reply:
Maybe `ECO_CI_INVENTORY_PATH`?